### PR TITLE
elijah, gavin pull request

### PR DESCRIPTION
**Pull request by leachg2**

### CODE REVIEW

File: t04_refactored.py [hunk 1]
```diff
@@ -194,9 +194,46 @@ def team_13_adv():
...
+        isDead = True
+        sleep(delay)
+
+    else:
+        print("You're still in the cave, and no closer to being out of it. Sucks to be you.")
+        sleep(delay)
+
+    if isDead:
+        numbertest = input("Pick a number 1-10 to try and survive.")
+        numbertest = float(numbertest)
```
Comment (sheggen) [inline t04_refactored.py:224]: You probably wanted an `int` here, not a `float`. What happens if they put in `10.001`?

File: t04_refactored.py [hunk 2]
```diff
@@ -194,9 +194,46 @@ def team_13_adv():
...
+
+    if 5 < numbertest <= 10:
+        not isDead
+        print("You got lucky this time.")
+
+    else:
+        print("Unlucky, you're dead. Very sad")
+        quit()
```
Comment (sheggen) [inline t04_refactored.py:232]: Use the `kill_if_dead(isDead)` function that I provided you with to murder your user.

File: t04_refactored.py [hunk 3]
```diff
@@ -194,9 +194,46 @@ def team_13_adv():
...
+    else:
+        print("You're still in the cave, and no closer to being out of it. Sucks to be you.")
+        sleep(delay)
+
+    if isDead:
+        numbertest = input("Pick a number 1-10 to try and survive.")
+        numbertest = float(numbertest)
+
+    if 5 < numbertest <= 10:
+        not isDead
```
Comment (sheggen) [inline t04_refactored.py:227]: This doesn't save isDead after flipping it:

`isDead = not isDead`

File: t04_refactored.py (hunk 1 above)
Comment (leachg2) [reply t04_refactored.py:224]: i fixed it, do i issue another pull request?

**Pull request closed without merging**